crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -872,9 +903,79 @@ impl LogsController {
             metadata_store_client,
             metadata_writer,
             async_operations: JoinSet::default(),
+            find_logs_tail_semaphore: Arc::new(Semaphore::new(1)),
         })
     }
 
+    pub fn find_logs_tail(&mut self) {
```

Comment:
We probably also want to run this `find_logs_tail` method when initially creating the `LogsController` to cover the case where we start retrieve the latest `Logs` configuration and don't see a newer version later.